src/store/index.js

```diff
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
 import modules from './modules'
-import ae from './aeppsdk'
+import EpochChain from '@aeternity/aepp-sdk/es/chain/epoch'
```

Comment:
please import as documented in the [README.md](https://github.com/aeternity/aepp-sdk-js/blob/develop/README.md): `import Aepp from '@aeternity/aepp-sdk/es/ae/aepp'`

Reply:
Maybe I am using it wrong. But using Aepp doesn't work. 
```
import Aepp from '@aeternity/aepp-sdk/es/ae/aepp'

Aepp({url: 'https://sdk-testnet.aepps.com'}).then(client => {
  client.height().then(height => {
    console.log('Current Block', height)
  })
})

```
Above code gives an error: `Uncaught TypeError: Cannot destructure property `resolve` of 'undefined' or 'null'.`

Whereas, the following works just fine:
```
import EpochChain from '@aeternity/aepp-sdk/es/chain/epoch'

EpochChain({url: 'https://sdk-testnet.aepps.com'}).then(client => {
  client.height().then(height => {
    console.log('Current Block', height)
  })
})
```